### ISSUE TITLE
Are you still maintaining this project?

### ISSUE THREAD

**Issue by marcofranssen** [emlun/gws2#15]
Given the last commit was 3 years ago. Are you still maintaining this project.

I have been using `gws` for years and was looking for another way.

If you have plans to maintain it, I would likely contribute in my plans to learn Rust.

**Comment by emlun** [comment]
Yeah, it's been a while since this was actively maintained, in part because I don't use it much myself anymore. But you're very welcome to contribute if you'd like to! I'll be happy to review any contributions you may have. I'll also take a look to make sure the project still builds and runs correctly.
